tests/test_synth.py

```diff
@@ -92,6 +92,18 @@ def test_deterministic_noise(self):
         assert torch.mean(torch.abs(x11 - x2)) > 1e-6
         assert torch.mean(torch.abs(x11 - x12)) < 1e-6
 
+        if self.device == "cuda":
```

Comment:
Give a comment or better names to the synths so this test is easier to understand?